src/apps/FEMSWEEP-OMP.cpp

```diff
@@ -0,0 +1,89 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "FEMSWEEP.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include <iostream>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+
+void FEMSWEEP::runOpenMPVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
+{
+#if defined(RAJA_ENABLE_OPENMP) && defined(RUN_OPENMP)
+
+  const Index_type run_reps = getRunReps();
+
+  FEMSWEEP_DATA_SETUP;
+
+  switch ( vid ) {
+
+    case Base_OpenMP : {
+
+      startTimer();
+      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+         #pragma omp parallel for
+         for (int ag = 0; ag < na * ng; ++ag)
+         {
+            FEMSWEEP_KERNEL;
+         }
+
+      }
+      stopTimer();
+
+      break;
+    }
+
+    case RAJA_OpenMP : {
+
+      auto res{getHostResource()};
+
+      using launch_policy =
+          RAJA::LaunchPolicy<RAJA::omp_launch_t>;
+
+      using outer_x =
+          RAJA::LoopPolicy<RAJA::omp_for_exec>;
+
+      startTimer();
+      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+         RAJA::launch<launch_policy>( res,
+             RAJA::LaunchParams(),
+             [=] RAJA_HOST_DEVICE(RAJA::LaunchContext ctx) {
+             RAJA::loop<outer_x>(ctx, RAJA::RangeSegment(0, na * ng),
+               [&](int ag) {
+                 FEMSWEEP_KERNEL;
+               });
+         });
+
+      }
+      stopTimer();
+
+      break;
+    }
+
+    default : {
+      getCout() << "\n FEMSWEEP : Unknown variant id = " << vid << std::endl;
```

Comment:
We are a bit inconsistent with the output here. Backends like OMPTarget and Sycl will mention the backend being used, for example:

```
      getCout() << "\n FEMSWEEP : Unknown OpenMPTarget variant id = " << vid << std::endl;

```

It may be useful information to add the backend being used?